### PR TITLE
Add a registerProjectBundles method in the AppKernel

### PR DESCRIPTION
**Pull request by fitn**
Here is a way to ease the registration of the project bundles. A dedicated method has been extracted and put at the top of the AppKernel.

### CODE REVIEW

File: app/AppKernel.php [hunk 1]
```diff
@@ -13,14 +13,22 @@
 class AppKernel extends Kernel
 {
     /**
-     * {@inheritdoc}
+     * Registers your custom bundles
      */
-    public function registerBundles()
+    protected function registerProjectBundles()
```
Comment (phaseinducer) [inline app/AppKernel.php]: public methods first ; )

**Comment by phaseinducer** [comment]
Godd idea, GTM

File: app/AppKernel.php [hunk 2]
```diff
@@ -13,14 +13,24 @@
 class AppKernel extends Kernel
 {
     /**
-     * {@inheritdoc}
+     * Registers your custom bundles
+     * 
+     * @return array
      */
-    public function registerBundles()
+    protected function registerProjectBundles()
```
Comment (phaseinducer) [inline app/AppKernel.php]: Public methods first ; )

**Comment by skeleton** [comment]
gtm

**Comment by nidup** [comment]
Thank you, great idea. Public method first, please squash your commit then GTM.

**Pull request merged**